core/src/main/java/tc/oc/pgm/score/ScoreModule.java

```diff
@@ -146,7 +152,13 @@ public ScoreModule parse(MapFactory factory, Logger logger, Document doc)
                   region, points, filter, ImmutableMap.copyOf(redeemables), silent));
         }
       }
-      return new ScoreModule(config, scoreBoxFactories.build());
+      return new ScoreModule(
+          new ScoreConfig(
+              scoreLimit.orElse(-1),
+              deathScore.orElse(proto.isOlderThan(MapProtos.DEFAULT_SCORES_TO_ZERO) ? 1 : 0),
+              killScore.orElse(proto.isOlderThan(MapProtos.DEFAULT_SCORES_TO_ZERO) ? 1 : 0),
+              mercyLimit.orElse(-1)),
```

Comment:
all of these "or else" will not trigger unless there was no xml tag for scores at all, otherwise they will be non-empty optionals, which defeats the purpose